docs/internals/contributing/committing-code.txt

```diff
@@ -32,8 +32,8 @@ alias will be helpful::
     [alias]
         pr = !sh -c \"git fetch upstream pull/${1}/head:pr/${1} && git checkout pr/${1}\"
 
-Add it to your ``~/.gitconfig``, and set ``upstream`` to be ``django/django``.
-Then you can run ``git pr ####`` to checkout the corresponding pull request.
+Add it to your ``~/.gitconfig``, and set ``upstream`` it to be ``django/django``.
+Then you can run ``git pr ####`` to check out the corresponding pull request.
```

Comment:
That's not a typo.

Reply:
Sorry, I got it the wrong way. Reverting it